tracing01-xdp-simple/trace_load_and_stats.c

```diff
@@ -23,11 +23,9 @@ static const char *__doc__ = "XDP loader and stats program\n"
 #include <linux/if_link.h> /* depend on kernel-headers installed */
 
 #include "../common/common_params.h"
-#include "../common/common_user_bpf_xdp.h"
-#include "../common/common_libbpf.h"
 
 #include <linux/perf_event.h>
-#define _GNU_SOURCE         /* See feature_test_macros(7) */
+#define _GNU_SOURCE		 /* See feature_test_macros(7) */
```

Comment:
Not sure what has happened to the whitespace in this file, but it's likely to annoy the maintainers who I think expect kernel whitespace rules to be followed.